scripts/workers/code_upload_submission_worker.py

```diff
@@ -557,31 +567,36 @@ def main():
         cluster_name, cluster_endpoint, challenge, evalai
     )
     install_gpu_drivers(api_instance)
+    api_instance = get_api_object(
```

Comment:
Why are we overriding `api_instance` here? It is already being used at multiple places

Reply:
@Ram81, it was being overridden in the While before and the parameters were not changing, so just moved the code outside the While. 
Also, it is assigned firstly by `get_api_client` and then calls `install_gpu_drivers` and then assigned by `get_api_object` so followed the same thing. Just took api_instance out of the while loop.
We could change the name of the firstly used api_instance.